### PR TITLE
Update TO changelog.md

### PR DESCRIPTION
**Pull request by ShahidHussain-srti**

### CODE REVIEW

File: content/sre/tenant-operator/changelog.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 ### Changes
 
+- fix: ResourceSupervisor CR no longer requires a field for the Tenant name
```
Comment (MuneebAijaz) [inline content/sre/tenant-operator/changelog.md:7]: version is wrong

Comment (ShahidHussain-srti) [reply content/sre/tenant-operator/changelog.md:7]: hmm

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-330-782bdb83`

**Review by abdulhaseeb2 — APPROVED** [review]

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.256`

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-330-b8ae7f6e`

**Pull request merged**